### DISCUSSION TITLE
Cordova Plugin Compatibility Problems with Android 14 After cordova-android 13 Update

### DISCUSSION THREAD

**Discussion by Topiya** [apache/cordova#491] [Q&A]
In Ionic 5 app, I upgraded **cordova-android to version 13** to target Android 14 (API level 34). 

However, after this update, the following plugins stopped working on **Android 14**:

- cordova-plugin-music-controls2 (v3.0.7)
- cordova-plugin-local-notification-12 (v0.1.3)
- cordova-plugin-sms-retriever-manager (v1.0.5)

Has anyone else experienced these issues after upgrading to cordova-android 13? Is there any known solution or workaround for this?

**Comment by breautek** [comment]
I'm moving this to Cordova Discussions since it doesn't describe a bug with Cordova-Android.

Cordova-android 12 -> 13 doesn't introduce any breaking changes involving surrounding the Cordova API. All breaking changes introduced are from the new tooling it requires for API 34 and changes that comes when targeting API 34 itself. These breaking changes can be found [here](https://developer.android.com/about/versions/14/behavior-changes-all). It will be up to the plugins to maintain support against breaking android changes.
